src/Generator/TwigToPdfGenerator.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\InvoicingPlugin\Generator;
+
+use Knp\Snappy\GeneratorInterface;
+use Symfony\Component\Config\FileLocatorInterface;
+use Twig\Environment;
+
+final class TwigToPdfGenerator implements TwigToPdfGeneratorInterface
```

Comment:
If I see correctly, you are not using this service anymore in this PR 👀

Reply:
Yes, it should be deleted.

Reply:
deleted now